core/services/feeds/proto/feeds_manager.pb.go

```diff
@@ -89,13 +87,11 @@ var (
 		0: "CHAIN_TYPE_UNSPECIFIED",
 		1: "CHAIN_TYPE_EVM",
 		2: "CHAIN_TYPE_SOLANA",
-		3: "CHAIN_TYPE_COSMOS",
```

Comment:
I think this was introduced recently in a cosmo add; but we'd need this to be added in FMS as well